content/nginxaas-azure/getting-started/prerequisites.md

```diff
@@ -8,7 +8,19 @@ type:
 - how-to
 ---
 
-Before you deploy F5 NGINX as a Service for Azure (NGINXaaS) you need to meet the following prerequisites:
+## NGINX Plus users
+
+If you are an existing F5 NGINX Plus user and you want to migrate to F5 NGINX as a Service for Azure (NGINXaaS), you need to know the following:
+
+- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates in some kind of storage and configure the network and subnet to connect to NGINX Plus.
```

Comment:
seems redundant

```suggestion
- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates and configure the network and subnet to connect to NGINX Plus.
```

Reply:
need to emphasize that the certificates are kept in a file storage system (as opposed to Azure Key Vault)

@mjang please suggest again

Reply:
Oh, one of those filesystem things.... maybe? 

```suggestion
- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates locally, and configure the network and subnet to connect to NGINX Plus.
```

Reply:
or maybe

```suggestion
- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates on that system, and configure the network and subnet to connect to NGINX Plus.
```

Reply:
Or maybe -- if you consider NFS drives

```suggestion
- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates in the filesystem, and configure the network and subnet to connect to NGINX Plus.
```

Reply:
I favor "file system"